plugins/catalog/src/js/repositories/components/RepositoriesTabUI.js

```diff
@@ -35,6 +35,8 @@ export default class RepositoriesTabUI extends React.Component {
 
     const { repositories, searchTerm, onSearch } = this.props;
 
+    console.log(repositories);
```

Comment:
:question:

Reply:
Oops, I must have accidentally committed something from when I was trying to learn about how the datalayer works :x